File: Code/Client/src/app/models/admin/EmpresaMantenimiento.ts
import { Basemodel } from "../base/Basemodel";

export class EmpresaMantenimiento extends Basemodel {
    NombreEmpresa?: string | null;
    NombreResponsable?: string | null;
    ApellidoResponsable?: string | null;
    Telefono?: string | null;
    Nit?: string | null;
    Direccion?: string | null;

    constructor(){
        super();
        this.NombreEmpresa = null;
        this.NombreResponsable = null;
        this.ApellidoResponsable = null;
        this.Telefono = null;
        this.Nit = null;
        this.Direccion = null;
    }


    
}

File: Code/Server/src/Core/Entities/EmpresaMantenimiento.cs
namespace IMT_Reservas.Server.Core.Entities;

public class EmpresaMantenimiento : Entity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Direccion { get; set; }
    public string? Telefono { get; set; }
    public string? NombreResponsable { get; set; }
    public string? ApellidoResponsable { get; set; }
    public bool EstadoEliminado { get; set; }
}
